src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
```

Comment:
The use of `String(robot[key])` is redundant here since object keys are automatically converted to strings. You can directly assign `invertedRobot[robot[key]] = key;`.